README.md

```diff
@@ -80,6 +80,44 @@ Import some deps into your app.scss:
 /* or just import the bits you need: @import 'foundation/functions'; */
 ```
 
+## Addon Usage
+
+To compile sass within an addon, there are a few additional steps:
+
+1. Include your styles in `addon/styles/addon.scss`.
+
+2. Ensure you've installed `ember-cli-sass` under `dependencies` in your
+   `package.json`. If you don't do this, it won't even try to compile
+   your sass.
+
+3. Define an `included` function in your app:
+   ```js
+   // in your index.js
+   module.exports = {
+     name: 'my-addon',
+
+     included: function(app) {
+       this._super.included(app);
```

Comment:
@aexmachina I'm using `ember-cli-sass": "4.0.0-beta.7` in an add on and can confirm that error is thrown without the documented include in the add ons `index.js`.